workbench/_api/routes/models.py

```
@@ -103,15 +105,22 @@ async def get_models(
         # Local models are fully loaded on the dev backend, so they're effectively hot.
         for model in models:
             model['status'] = ModelHeat.HOT.value
-        return models
+        
+    ## JLens supported models
+    try:
+        lens_models = j_lens.get_available_lenses()
+        for model in models:
+            name = model.get("name", "")
+            model["has_jacobian"] = name.rsplit("/", 1)[-1] in lens_models
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)